bahmni-lite/docker-compose.yml

```diff
@@ -279,6 +279,12 @@ services:
     # volumes:
     # - "${APPOINTMENTS_PATH:?}/dist/:/usr/local/apache2/htdocs/appointments"
 
+  sms-service:
+    image: bahmni/sms-service:${SMS_SERVICE_IMAGE_TAG:?}
+    profiles: [ "emr" ]
```

Comment:
Lets not put this under `emr` profile as this profile should be the bare minimum set of services used to explore/try emr with lowest resources. We can create a new profile `sms` and document it on the steps to enable it

Reply:
agree

Reply:
updated profile